dcos-log/journal/reader/read_test.go

```diff
@@ -195,3 +198,71 @@ func TestOptionMatchOR(t *testing.T) {
 		t.Fatalf("Expecting to find 2 entries, got %d", size)
 	}
 }
+
+func TestFollow(t *testing.T) {
```

Comment:
TestFollow looks like a good test for the happy path of the `Follow()` method, but it doesn't seem to test the main purpose of the Follow method which handles `SD_JOURNAL_INVALIDATE`.   Is this something that's too difficult to test in the unit tests?  I wonder if this could be tested in the integration tests...